lib/mocha/cardinality.rb

```diff
@@ -96,6 +93,12 @@ def count(number)
       end
     end
 
+    def quantify(required, maximum)
+      @required = required
+      @maximum = maximum
+      self
```

Comment:
Do we definitely need to return `self` here? The method chaining is all at the `Expectation` level, so I would've thought it would be the methods on that class which would return `self`, no?

Reply:
I see. I read the code too quickly and assumed that returning `self` in this method was connected to the changes to the methods in `Expectation`, but I now realise they are not connected.